src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
+
+export type TodoStatus = 'All' | 'Active' | 'Completed';
+
+type Props = {
+  todos: Todo[];
+  status: TodoStatus;
+  setStatus: (status: TodoStatus) => void;
+};
+
+export const Footer: React.FC<Props> = ({ todos, status, setStatus }) => {
+  const remainingTodos = todos.filter(todo => !todo.completed).length;
+  const completedTodos = todos.some(todo => todo.completed);
```

Comment:
```suggestion
  const hasCompletedTodos = todos.some(todo => todo.completed);
```
Boolean variables should start from `is` or `has`